src/Components/Footer/Footer.css

```
@@ -3,3 +3,64 @@
   height: 5rem;
   background-color: var(--midBlue-color);
 }
+
+.footer-content-container {
+  margin: auto;
+  max-width: var(--main-max-width);
+  padding: 2rem 1rem;
+  display: flex;
+  justify-content: space-between;
+}
+
+.footer-logo {
+  height: 2rem;
+  margin-bottom: 1rem;
+}
+
+.white-font {
+  color: #ffff;
```

You are saying that the opacity is 100% with this one.
```suggestion
  color: #ffffff;
```